primitive/zk_rollup/src/operator.rs

```diff
@@ -7,13 +7,27 @@ use crate::{
     poseidon::FieldHasher,
     proof::Proof,
 };
-#[cfg(test)]
 use red_jubjub::PublicKey;
 
-#[derive(Debug, PartialEq)]
-pub(crate) struct Batch<F: FftField, H: FieldHasher<F, 2>, const N: usize, const BATCH_SIZE: usize>
+pub trait BatchGetter<F: FftField> {
```

Comment:
what's BatchGetter?
This would be better to integrate with some other trait

Reply:
It's just a trait to provide an access to Batch type inside of a pallet.